src/app/static/src/tests/integration/surveyAndProgram.itest.ts

```diff
@@ -25,8 +26,11 @@ describe("Survey and program actions", () => {
 
         expect(commit.mock.calls[0][0]).toStrictEqual({
             type: "SurveyError",
-            payload: "could not find function \"validate_func\""
+            payload: "File does not exist. Create it, or fix the path."
         });
+
+        //expectSelectedDataTypeUpdate(commit.mock.calls[1], DataType.Survey);
```

Comment:
These can be reinstated once we start getting 200s - once merged with the volume mount?